packages/synapse-react-client/test/lib/containers/personal_access_token/CreateAccessTokenModal.test.tsx

```diff
@@ -58,47 +75,61 @@ describe('CreateAccessTokenModal tests', () => {
     )
 
     // Close the modal using the 'Close' button
-    await userEvent.click(
-      (
-        await screen.findAllByRole('button', { name: 'Close' })
-      )[1],
-    )
+    await user.click(screen.getByRole('button', { name: 'Close' }))
```

Comment:
Previous modal had two buttons named 'Close' - the button itself and the "X" in the header. ConfirmationDialog only has one button named 'Close". The  "X" in the header is named 'close'.